api/types/events/events.proto

```diff
@@ -1673,6 +1673,26 @@ message WindowsDesktopSessionEnd {
     map<string, string> DesktopLabels = 8 [ (gogoproto.jsontag) = "desktop_labels" ];
 }
 
+message UserCertsEmit {
```

Comment:
Let's avoid abbreviations in public protos / API surface.

```suggestion
message UserCertificateEmitted {
```

Could be `UserCertificateIssued` or something to that effect as well. Use plural if a single event would count for multiple certificates (although I'd rather have one event per cert).

Reply:
I always talked in terms of "certificates" or "certificate set" because user certs are always generated in a pair of SSH certificate and X.509 certificate - should that be considered a singular "certificate"?